patches/minecraft/net/minecraft/entity/player/EntityPlayer.java.patch

```diff
@@ -520,6 +531,11 @@
 +    }
 +
 +    private String displayname;
++    
++    /** A collection of components to prepend to the username. This is not persisted across restarts */
```

Comment:
Figured it'd be cleaner to have it in `EntityPlayer` rather than keeping a map between players and prefixes/suffixes in `ForgeHooks` or some other class.